sklearn/decomposition/_fastica.py

```diff
@@ -527,7 +530,6 @@ def g(x, fun_args):
             )
 
         n_features, n_samples = XT.shape
-
```

Comment:
Note for later: it's better not to introduce formatting changes that are unrelated to the lines edited by the PR. Otherwise, the diff can quickly get crowded by unrelated changes that makes the review process more difficult.

It's not a problem for this specific PR because the diff is small but more a general remark for future contribution.

Reply:
Ok thanks ! I'll keep that in mind for the future :)